### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by oorjadandotiya**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -267,3 +267,10 @@
   title: Here I come Coding In
   img: haider2.jpg
   author_name: Haider Shafaq
+
+- url: https://technoorja.wordpress.com/2018/01/01/technoorja-learns-to-code/
+  title: Technoorja Learns To Code
+  img: Oorja_modified.jpg
+  author_name: Oorja Dandotiya
+- url: https://technoorja.wordpress.com/2018/01/01/technoorja-learns-to-code/
+  title
```
Comment (nikit19) [inline _data/blogs.yml:276]: Please use the exact format as given in the README.md 
You have written url twice

File: _data/blogs.yml [hunk 2]
```diff
@@ -267,3 +267,10 @@
   title: Here I come Coding In
   img: haider2.jpg
   author_name: Haider Shafaq
+
+- url: https://technoorja.wordpress.com/2018/01/01/technoorja-learns-to-code/
+  title: Technoorja Learns To Code
+  img: Oorja_modified.jpg
```
Comment (nikit19) [inline _data/blogs.yml:273]: Please upload this image in the img/blogs folder

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please make these changes

File: _data/blogs.yml (hunk 2 above)
Comment (oorjadandotiya) [reply _data/blogs.yml:273]: I have uploaded that image

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Upload related picture to `img`

**Comment by akshatnitd** [comment]
Closing this PR, since a duplicate PR is already open in #785

**Pull request closed without merging**